src/ovp_pipeline/commands/refresh_entities.py

```
@@ -0,0 +1,219 @@
+        "refreshed": refreshed,
+        "refreshed_count": len(refreshed),
+        "orphan_files": orphan_files,
+        "orphan_count": len(orphan_files),
+        "missing_files": missing_files,
+        "missing_count": len(missing_files),
+        "type_mismatches": type_mismatches,
+        "dry_run": dry_run,
+        "fix_applied": fix and not dry_run,
+    }
+
+    print(f"Entity Refresh Report:")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)